go.mod

```diff
@@ -13,6 +13,7 @@ require (
 	github.com/spf13/viper v1.7.0
 	golang.org/x/crypto v0.0.0-20200728195943-123391ffb6de
 	golang.org/x/sync v0.0.0-20200625203802-6e8e738ad208
+	gopkg.in/check.v1 v1.0.0-20200227125254-8fa46927fb4f
```

Comment:
I am not sure why is this dependency added. Probably by some auto import. Go mod tidy should help.